src/makeCalculator.js

```diff
@@ -5,6 +5,29 @@
  */
 function makeCalculator() {
   // write code here
+  let result = 0;
+
+  const calculator = {
+    add: (num) => result + num,
+    subtract: (num) => result - num,
+    divide: (num) => result / num,
+    multiply: (num) => result * num,
+    operate: function (callback, num) {
+      result = callback(num);
+
+      return this;
+    },
+    reset: function () {
+      result = 0;
+
+      return this;
+    },
```

Comment:
[CODE KNOWLEDGE] - When defining methods within an object literal, you can use the shorthand syntax without the `function` keyword. For example, use `operate(callback, num) { ... }` and `reset() { ... }` instead of using the `function` keyword.